### PR TITLE
profiles: move read-only config entries to disable-common.inc

### PR DESCRIPTION
**Pull request by kmk3**
This should reduce duplication and increase hardening a bit.

Relates to #1642 #5706 #5707 #5710.

Cc: @pirate486743186 (from #5706 #5707 #5710)

### CODE REVIEW

File: etc/profile-a-l/firefox.profile [hunk 1]
```diff
@@ -66,5 +66,7 @@ dbus-user.own org.mpris.MediaPlayer2.firefox.*
 #ignore noroot
 ignore dbus-user none
 
+read-write ${HOME}/.mozilla/firefox/profiles.ini
```
Comment (rusty-snake) [inline etc/profile-a-l/firefox.profile:69]: Overridden by later read-only.

**Review by rusty-snake — DISMISSED** [review]

File: etc/profile-a-l/firefox.profile (hunk 1 above)
Comment (kmk3) [reply etc/profile-a-l/firefox.profile]: > Overridden by later read-only.

Good catch; fixed.

File: etc/templates/profile.template [hunk 2]
```diff
@@ -221,6 +221,8 @@ include globals.local
 #dbus-user.talk org.freedesktop.Notifications
 #dbus-system none
 
+# Note: read-only entries should usually go on disable-common.inc (especially
```
Comment (glitsj16) [inline etc/templates/profile.template:224]: Typo: ... go in disable-common.inc ...

**Review by glitsj16 — APPROVED** [review]
LGTM. A single typo change and we can bring this in IMO.

File: etc/templates/profile.template (hunk 2 above)
Comment (kmk3) [reply etc/templates/profile.template]: > Typo: ... go in disable-common.inc ...

Changed.

By the way, are you sure that there is a preferred form?

I think I've seen it used both ways, so I don't know.

Comment (glitsj16) [reply etc/templates/profile.template]: > By the way, are you sure that there is a preferred form?

Sorry, I don't understand what preferred form you're referring to in this context.

Comment (glitsj16) [reply etc/templates/profile.template]: Oh, I just thought it was an actual typo. If you'd like to keep the "on $file" form, please disregard my nitpick.

**Comment by netblue30** [comment]
all merged!

**Pull request merged**